jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/model/CodeWhispererModel.kt

```diff
@@ -21,21 +21,21 @@ data class CaretContext(val leftFileContext: String, val rightFileContext: Strin
 data class FileContextInfo(
     val caretContext: CaretContext,
     val filename: String,
-    val programmingLanguage: ProgrammingLanguage
+    val programmingLanguage: CodeWhispererProgrammingLanguage
 )
 
-data class ProgrammingLanguage(private val myLanguageName: String) {
-    // case-insensitive
-    val languageName get() = myLanguageName.lowercase()
-    constructor(codewhispererLanguage: CodewhispererLanguage) : this(codewhispererLanguage.toString())
-
-    override fun equals(other: Any?): Boolean {
-        if (other !is ProgrammingLanguage) return false
-        return other.languageName == this.languageName
-    }
-
-    override fun toString(): String = languageName
-}
+// data class ProgrammingLanguage(private val myLanguageName: String) {
+//    // case-insensitive
+//    val languageName get() = myLanguageName.lowercase()
+//    constructor(codewhispererLanguage: CodewhispererLanguage) : this(codewhispererLanguage.toString())
+//
+//    override fun equals(other: Any?): Boolean {
+//        if (other !is ProgrammingLanguage) return false
+//        return other.languageName == this.languageName
+//    }
+//
+//    override fun toString(): String = languageName
+// }
```

Comment:
Is this still needed?

Reply:
oops nope, I should have removed it, will update